services/uptimerobot/uptimerobot-base.js

```diff
@@ -74,6 +74,7 @@ export default class UptimeRobotBase extends BaseJsonService {
           ...opts,
         },
       },
+      logErrors: [],
```

Comment:
See https://github.com/badges/shields/issues/9155 for details.

The rate limits we're hitting here are on the user-supplied tokens. There's not really anything we can do about this.